db/migrate_cards/data/cards/homepage_company_metric_item

```diff
@@ -1,25 +1,22 @@
-
+  
   <!--prototype: Company+MetricDesigner+MetricName+yinyang drag item -->
   <div class="metric-item value-item">
     <div>
+      <a class="inherit-anchor" href= "/{{_1|linkname}}?filter[metric]={{_llr+_lr|linkname}}">

```

Comment:
as in `{{_llr+_lr|cgi_escape_name}}`

we might want to alias that with something shorter, like `cgi_name`